client/console/src/main/java/org/apache/syncope/client/console/init/ClassPathScanImplementationLookup.java

```diff
@@ -161,5 +169,15 @@ public int compare(
     public List<Class<? extends BaseExtWidget>> getExtWidgetClasses() {
         return extWidgets;
     }
+    
+    /**
+     * @return basePackages for syncope class classpath scanning
+     */
+    protected List<String> getBasePackages() {
```

Comment:
Did not get what this method was for, at a first glance, then I've realized it might be used for subclasses: please mention this in the javadoc.